db/schema.rb

```diff
@@ -12,7 +12,7 @@
 
```

Comment:
Maybe just don't commit these schema changes. This probably was caused by your local mysql. just do not include them in the commit